exec/java-exec/src/main/java/org/apache/drill/exec/client/DrillClient.java

```diff
@@ -223,19 +223,65 @@ public void connect(Properties props) throws RpcException {
     connect(null, props);
   }
 
+  /**
+   * Function to populate the endpointList with information of all the drillbits
+   * provided in the connection string by client
+   * @param endpointList - ArrayList of DrillbitEndpoints
+   * @param drillbits - One or more drillbit ip[:port] provided in connection string
```

Comment:
One or more drillbit -> drillbits

(Javadoc practice is to use lower case in param comments. See: http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#styleguide)

Please copy detailed description from method body to here so it shows up in Javadoc.

I one port allowed? Or, should the user specify all three? If only one, what port numbers are used for the other two? If this is documented elsewhere, just point to that location.